tools/idl_interfaces.sh

```diff
@@ -16,20 +16,25 @@ function generate_core_interface {
     CORE_DIR=$1 # Ledger Core library directory
     CORE_IDL_DIR=$CORE_DIR/idl
     CORE_API_DIR=$CORE_DIR/inc/core/api
+    CORE_CPP_JNI_DIR=$CORE_DIR/inc/core/jni
 
     echo -e "Generating ledger-core API"
 
     # recreate API directory
     rm -rf $CORE_API_DIR
     mkdir -p $CORE_API_DIR
 
-    ./djinni/src/run \
+    ../djinni/src/run \
```

Comment:
`HODL` until https://github.com/LedgerHQ/djinni/pull/43 is merged and this is reverted to `./…`.